projects/ion/src/lib/modal/component/modal.component.ts

```diff
@@ -34,6 +35,7 @@ export class IonModalComponent implements OnInit, OnDestroy {
   @Input() configuration: IonModalConfiguration = {};
 
   @Output()
+  ionOnHeaderButtonAction = new EventEmitter<SafeAny>();
```

Comment:
```suggestion
  ionOnHeaderButtonAction = new EventEmitter<IonModalResponse | undefined>();
```
Couldn't this be typed as the value that will be emitted?

Reply:
yep, forgot to change this one